src/components/pages/Thanks.tsx

```diff
@@ -28,6 +30,36 @@ export const Thanks: React.FC = () => {
     <>
       <br />
       <Container>{JSON.stringify(data)}</Container>
+      <Card style={{ width: '18rem' }}>
+    <Card.Body>
+      <Media>
+      <img
+        width={64}
```

Comment:
Consider mapping over the data fetched from the cloud function 
to generate the view with real data.